Xamarin.Forms.ControlGallery.Android/FormsAppCompatActivity.cs

```diff
@@ -43,7 +43,9 @@ protected override void OnCreate(Bundle bundle)
 #if TEST_EXPERIMENTAL_RENDERERS
 			Forms.SetFlags("FastRenderers_Experimental");
 #else
-			Forms.SetFlags("Fake_Flag"); // So we can test for flag initialization issues
+			// Fake_Flag is here so we can test for flag initialization issues
+			// CollectionView lets us test CollectionView stuff until it's officially released
+			Forms.SetFlags("Fake_Flag", "CollectionView_Experimental"); 
```

Comment:
Why are these two grouped together as opposed to being separate statements where the latter part goes under `#if TEST_EXPERIMENTAL_RENDERERS`?

Reply:
Good point; I've added CollectionView to the "experimental" list.

Reply:
Would `CollectionView_Experimental` still be part of `Forms.SetFlags("Fake_Flag", "CollectionView_Experimental");`?

Reply:
Yes.  `TEST_EXPERIMENTAL_RENDERERS` is just there to allow our CI to build and UI test a version of Control Gallery with the fast renderers (a better name for it would have been `TEST_FAST_RENDERERS`).  CollectionView should be available for testing in both versions of Control Gallery.